applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -52,12 +52,9 @@ def AssembleTestSuites():
     smallSuite = suites['small']
     smallSuite.addTest(CouetteFlowTest('testCouetteFlow2DSymbolicStokes'))
     smallSuite.addTest(CouetteFlowTest('testCouetteFlow2DWeaklyCompressibleNavierStokes'))
-    smallSuite.addTest(EmbeddedCouetteTest('testEmbeddedCouette2D'))
-    smallSuite.addTest(EmbeddedCouetteTest('testEmbeddedSlipCouette2D'))
-    smallSuite.addTest(EmbeddedCouetteTest('testEmbeddedAusasCouette2D'))
-    smallSuite.addTest(EmbeddedCouetteTest('testEmbeddedDevelopmentCouette2D'))
-    smallSuite.addTest(EmbeddedCouetteTest('testEmbeddedSlipDevelopmentCouette2D'))
-    smallSuite.addTest(EmbeddedCouetteTest('testEmbeddedAusasDevelopmentCouette2D'))
```

Comment:
These are old tests which are removed ryt?

Reply:
Correct. I arranged them in the embedded Couette flow test. The old elements are no longer tested while the "official" ones now use a more standard analysis-based test mechanism.